src/routes/pois/components/PoiListItem.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from 'react'
+import ListItem from '../../../modules/common/components/ListItem'
+import PoiModel from '../../../modules/endpoint/models/PoiModel'
+
+type PropsType = {|
+  poi: PoiModel
+|}
+
+class PoiListItem extends React.Component<PropsType> {
+  render () {
+    const {poi} = this.props
+    return (
+      <ListItem key={poi.path}
+                thumbnail={poi.thumbnail}
```

Comment:
How does it react, if some pois have a thumbnail and some don't?
Are the items unaligned then?